src/main/resources/db/migration/V002_2__Create_user_messages_table.sql

```diff
@@ -0,0 +1,5 @@
+create table MESSAGES (
+    ID int not null,
+    USER_ID int not null,
+    MESSAGES  varchar(200)
```

Comment:
liczba mnoga w nazwie kolumny raczej nie pojawia się - a jak juz to bardzo rzadko